libs/application/templates/general-fishing-license/src/fields/Declined/WarningSection.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Application,
+  formatText,
+  coreMessages,
+} from '@island.is/application/core'
+import { useLocale } from '@island.is/localization'
+import React, { FC } from 'react'
+import { Box, Icon, Tag, Text } from '@island.is/island-ui/core'
+import * as styles from './WarningSection.css'
+
+interface Step {
+  key: string
+  title: string
+  description: string
+  state: boolean
+}
+
+type WarningSectionProps = {
+  application: Application
+  step: Step
+}
+
+export const WarningSection: FC<WarningSectionProps> = ({
+  application,
+  step: { description, title },
+}) => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <Box
+      position="relative"
+      border="standard"
+      borderRadius="large"
+      padding={4}
+      marginBottom={2}
+      borderColor={'red600'}
+    >
+      {/* Warning Icon */}
```

Comment:
wouldnt really say that these comments are needed.

Reply:
Removed them